plugin/gv.vim

```diff
@@ -297,14 +297,19 @@ function! s:gl(buf, visual)
   call setloclist(0, insert(getloclist(0), {'bufnr': a:buf}, 0))
   b #
   lopen
+  let title = 'o: open / o (in visual): diff / O: open (tab) / q: quit'
+  try
+    call setloclist(0, [], 'a', {'title': title})
+  catch
```

Comment:
Instead of catching everything:

    catch /^Vim\%((\a\+)\)\=:E118/	" Error: Too many arguments for function

(Although neither case sets the title for me on vim 7.4.52.)

Reply:
Yeah, that is outdated.

I've improved list behavior in Neomake lately, but do not plan to do it here / update this.

Closing, but feel free to pick it up, of course.